client/src/app/components/deadline-counter/deadline-counter.component.ts

```diff
@@ -33,9 +34,9 @@ export class DeadlineCounterComponent implements OnInit {
     }
     private startCounter(){
         this.interval = setInterval(()=>{
-            const now = new Date().getTime() - 100000;
-            const countDownDate = this.date.getTime();
-            var distance = countDownDate - now;
+            const now = moment();
+            const date = moment(this.date);
+            var distance = date.diff(now);
             this.daysLeft = Math.floor(distance / (1000 * 60 * 60 * 24));
             this.hoursLeft= Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
             this.minutesLeft = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
```

Comment:
You can easily calculate days, hour, minutes difference with https://momentjs.com/docs/#/displaying/difference/ passing as second argument the unit. Please check docs and change, eg. `daysLeft = date.diff(now, 'days')`

Reply:
Changed